src/Models/ContactBase.js

```diff
@@ -0,0 +1,51 @@
+import declare from 'dojo/_base/declare';
+import _ModelBase from 'argos/Models/_ModelBase';
+import MODEL_NAMES from './Names';
+
+const resource = window.localeContext.getEntitySync('contactModel').attributes;
+
+const __class = declare('crm.Models.ContactBase', [_ModelBase], {
+  resourceKind: 'contacts',
+  entityName: 'Contact',
+  entityDisplayName: resource.entityDisplayName,
+  entityDisplayNamePlural: resource.entityDisplayNamePlural,
+  modelName: MODEL_NAMES.CONTACT,
+  iconClass: 'fa fa-user fa-lg',
+  security: 'Entities/Contact/View',
+  createRelationships: function createRelationships() {
+    let rel;
+    rel = this.relationships || (this.relationships = [{
+      name: 'Addresses',
+      displayName: 'Addresses',
+      propertyName: 'Addresses',
+      type: 'OneToMany',
```

Comment:
We should probably get a task in to make these some sort of enum type instead of a magic value.